packages/rest/src/lib/utils/constants.ts

```diff
@@ -31,6 +31,7 @@ export const DefaultRestOptions = {
 	async makeRequest(...args): Promise<ResponseLike> {
 		return getDefaultStrategy()(...args);
 	},
+	mediaProxy: 'https://media.discordapp.net',
```

Comment:
lets get this in -types too

Reply:
https://github.com/discordjs/discord-api-types/pull/1001